Applications/Hermes/hermesflaskserver.py

```diff
@@ -0,0 +1,177 @@
+import os
+import hermesobject as hermesobject
+import ConfigParser
+import json
+import base64
+import cPickle
+
+from flask import Flask
+from flask import request
+
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("hermes.ini")
+config.read(filename)
+
+# Global variables
+dictmsgobj = {}
+
+
+def configparse():
+    """ Parses configuration file for callsigns and node ID and returns them in JSON format"""
+    local = {}
+    num = int(config.get('PROXY', 'UNITS'))
+    units = range(0, num)
+
+    for unit in units:
+        # TODO We don't really check for valid input here yet
+        item = "UNIT" + str(unit)
+        callsign = config.get(item, "callsign").upper()
+        nodeid = config.get(item, "nodeid")
+
+        local[str(item)] = \
+            {
+                "callsign": callsign,
+                "nodeid": nodeid,
+            }
+
+    local = json.dumps(local)
+    return json.loads(local)
```

Comment:
wait, why dumps local then loads local? you're back with the same thing in the end? This just returns a dictionary... Actually a list of dictionarie(s) I believe.